nixos/modules/services/networking/monero.nix

```diff
@@ -64,6 +68,23 @@ in
         '';
       };
 
+      banlist.enable = lib.mkOption {
+        type = lib.types.bool;
+        default = true;
```

Comment:
This should definitely not be enabled by default.

Reply:
Once we require to provide banlist manually then yeah, but do you have any idea what would be a good idea to warn all users of this module to enable it? warn on every rebuild when something like services.monero.banlist.acknowledge is false?

again,

> The Monero Research Lab (MRL) has decided to recommend that **all** Monero node operators enable a ban list of suspected spy node IP addresses.